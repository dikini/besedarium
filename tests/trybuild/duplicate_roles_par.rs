use besedarium::*;

type DupRolePar = tpar!(Http;
    TInteract<Http, TClient, Message, TEnd<Http>>,
    TInteract<Http, TClient, Publish, TEnd<Http>>
);
assert_disjoint!(par DupRolePar);
