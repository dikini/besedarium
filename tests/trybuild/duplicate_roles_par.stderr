error[E0601]: `main` function not found in crate `$CRATE`
 --> tests/trybuild/duplicate_roles_par.rs:7:34
  |
7 | assert_disjoint!(par DupRolePar);
  |                                  ^ consider adding a `main` function to `$DIR/tests/trybuild/duplicate_roles_par.rs`
